src/Template.js

```diff
@@ -212,7 +212,7 @@ class Template extends TemplateContent {
 				let results = await Promise.all(promises);
 
 				permalinkValue = {};
-				for (let j = 0, k = keys.length; j < k; j++) {
+				for (let j = 0; j < keys.length; j++) {
```

Comment:
question: Extracting the length into a variable used to be a performance improvement thing. Is that irrelevant now with newer browser versions?